tests/Database/MutationValidationUniqueWithCustomRulesTests/MutationWithCustomRuleWithRuleObject.php

```diff
@@ -13,12 +13,12 @@ class MutationWithCustomRuleWithRuleObject extends Mutation
         'name' => 'mutationWithCustomRuleWithRuleObject',
     ];
 
-    public function type()
+    public function type(): Type
     {
         return Type::nonNull(Type::string());
     }
 
-    public function rules(array $args = [])
+    protected function rules(array $args = []): array
```

Comment:
There are a few visibility changes on `rules` but they're actually a correction because the parent has been `protected` all the time